Git-egylet/pacman.py

```diff
@@ -0,0 +1,254 @@
+
+import cv2
+import numpy as np
+from typing import Tuple
+
+
+class Pacman:
+    def __init__(self, world_size: int, max_steps: int, show_window_size: int) -> None:
+        """
+        This function initializes the base attributes of the Pacman class
+        :param world_size: size of the map
+        :param max_steps: maximum amount of steps during the game
+        :param show_window_size: size of the image in pixels
+        """
+        self.world_size = world_size
+        self.max_steps = max_steps
+        self.show_window_size = show_window_size
+        # map components list
+        self.body = []
+        self.walls = []
+        self.ghosts = []
+        self.pellets = []
+        # default values
+        self.step_counter = 0
+        self.direction = 1
+        self.score = 0
+        self.num_pellets = 10
```

Comment:
Ez is mehetne konstruktor argumentumba, ahogy a többi amit tök jól csináltál!
```suggestion
        self.num_pellets = num_pellets
```

Reply:
Solved